src/api/handlers/guild.ts

```diff
@@ -920,3 +922,69 @@ export async function editGuildTemplate(
   ) as GuildTemplate;
   return structures.createTemplate(template);
 }
+
+function createMembershipStruct(
```

Comment:
this name couldbe confused with `structures` and i think we should choose a different name

Reply:
Any suggestions?